README.md

```diff
@@ -1,49 +1,60 @@
-jbi.vim
-=======
+## motoman-inform-vim-plugin
 
-Vim syntax file for Motoman INFORM III.
-----------------------------------------------------------------------
+## [Vim][1] syntax and filetype plugin for [Motoman][2] INFORM III.
 
 ### Installation
 
-Copy jbi.vim to `~/.vim/syntax/` or `$VIMRUNTIME/syntax/`.
-Use the command
+#### Installation with vim-plug:
 
-```vim
-    :set ft=jbi
-```
+Put this in your .vimrc:
 
-to enable syntax highlighting for jbi files.
+    call plug#begin('~/.vim/plugged')
+      Plug 'matthijsk/motoman-inform-vim-syntax'
+    call plug#end()
 
-### Automatic filetype detection
+For the first installation run:
 
-You might want to add the following lines to `~/.vim/filetype.vim` to enable
-automatic detection of the jbi filetype:
-    
-```vim
-    " Detect INFORM III filetype.
-    augroup filetypedetect
-      au BufRead,BufNewFile *.jbi		setfiletype jbi
-      au BufRead,BufNewFile *.JBI		setfiletype jbi
-    augroup END
-```
+    :PlugInstall
 
-### Navigating jobs
+Update every once in a while with:
 
-It's nice to be able to quickly navigate through the source code, especially
-if you make heavy use of `CALL JOB:<JOBNAME>` statements. When editing a C or
-C++ file with Vim the current file under the cursor can be opened using `gf`
-or `CTRL-W_CTRL-F`. To enable this for .jbi files add the following to your
-.vimrc:
+    :PlugUpdate
+
+#### Manual installation
+
+Extract and copy the folders 
+`/doc`, `/ftdetect`, `/ftplugin`, `/indent` and `/syntax` 
```

Comment:
There are no `doc` and `indent` folders in this plugin.

Reply:
I don't know enough about this language to create an indent file.
Ans as said, I don't see a reason for more documentation apart from the readme. Maybe one can just rename it to jbi.txt and put it in the doc folder. But then one would have to take care about both files at once. If there was an doc/jbi.txt it would be possible to reach it by :help jbi.txt or :help jbi<tab> from within Vim, but is this worth it?
So I deleted this. Thanks